daml-lf/spec/daml-lf-1.rst

```diff
@@ -702,7 +703,7 @@ available for usage::
             , 'choices' { ChDef₁, …, ChDefₘ }
             , KeyDef
             }
-       |  'exception' (x: T)                        -- DefException [DAML-LF ≥ 1.dev]
+       |  'exception' T                             -- DefException [DAML-LF ≥ 1.dev]
```

Comment:
We need to add the `message` here as well, don't we?

Reply:
Indeed.